DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h

```diff
@@ -387,6 +387,8 @@ struct RecoContainer {
   }
   auto getITSTPCTRDTracksMCLabels() const { return getSpan<o2::MCCompLabel>(GTrackID::ITSTPCTRD, MCLABELS); }
   auto getITSTPCTRDTrackMCLabel(GTrackID id) const { return getObject<o2::MCCompLabel>(id, MCLABELS); }
+  auto getITSTPCTRDSATracksMCLabels() const { return getSpan<o2::MCCompLabel>(GTrackID::ITSTPCTRD, MCLABELSEXTRA); }
+  auto getITSTPCTRDSATraclMCLabel(GTrackID id) const { return getObject<o2::MCCompLabel>(id, MCLABELSEXTRA); }
```

Comment:
a small typo: getITSTPCTRDSATraclMCLabel -> getITSTPCTRDSATracklMCLabel

Reply:
Thanks, I fixed that and squashed the commits for the separate TRD MC labels